cmd/helm/init.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 const initDesc = `
-This command sets up local configuration in $HELM_HOME (default ~/.helm/).
+This command sets up local configuration in $HELM_HOME (default $XDG_CONFIG_DIR/helm/).
```

Comment:
same here; can we list out what exactly is usually $XDG_CONFIG_DIR? Many people know about $HOME, but not much about $XDG_CONFIG_DIR.

> This command sets up local configuration in $HELM_HOME (default $XDG_CONFIG_DIR/helm/). Based on your OS, $XDG_CONFIG_DIR is 

- `%APPDATA%\helm` on Windows
- `$HOME/Library/Preferences` on OSX
- `$HOME/.config/helm` on Linux

Reply:
Updated and fixed according to requests